topography/topography.F90

```diff
@@ -787,91 +791,727 @@ subroutine interp_topog_2d ( blon, blat, zout, flag )
     call horiz_interp_del ( Interp )
 
  end subroutine interp_topog_2d
+#else
+
+ function get_topog_mean_1d (blon, blat, zmean)
+
+   real, intent(in),  dimension(:)   :: blon, blat
+   real, intent(out), dimension(:,:) :: zmean
+   logical :: get_topog_mean_1d
+
+!-----------------------------------------------------------------------
+   if (.not. module_is_initialized) call topography_init()
+
+   if ( any(shape(zmean(:,:)) /= (/size(blon(:))-1,size(blat(:))-1/)) ) &
+        call error_mesg('get_topog_mean_1d','shape(zmean) is not&
+            & equal to (/size(blon)-1,size(blat)-1/))', FATAL)
+
+   if ( open_topog_file(topog_file) ) then
+       call interp_topog_1d ( blon, blat, zmean )
+       get_topog_mean_1d = .true.
+   else
+       get_topog_mean_1d = .false.
+   endif
+
+!-----------------------------------------------------------------------
+
+ end function get_topog_mean_1d
+
+!############################################################
+
+ function get_topog_mean_2d (blon, blat, zmean)
+
+   real, intent(in),  dimension(:,:) :: blon, blat
+   real, intent(out), dimension(:,:) :: zmean
+   logical :: get_topog_mean_2d
+
+!-----------------------------------------------------------------------
+   if (.not. module_is_initialized) call topography_init()
+
+   if ( any(shape(zmean(:,:)) /= (/size(blon,1)-1,size(blon,2)-1/)) .or. &
+        any(shape(zmean(:,:)) /= (/size(blat,1)-1,size(blat,2)-1/)) ) &
+        call error_mesg('get_topog_mean_2d','shape(zmean) is not&
+            & equal to (/size(blon,1)-1,size(blon,2)-1/))', FATAL)
+
+   if ( open_topog_file(topog_file) ) then
+       call interp_topog_2d ( blon, blat, zmean )
+       get_topog_mean_2d = .true.
+   else
+       get_topog_mean_2d = .false.
+   endif
 
+!-----------------------------------------------------------------------
+
+ end function get_topog_mean_2d
+
+! </FUNCTION>
 !#######################################################################
 
- subroutine find_indices ( ybeg, yend, ydat, js, je )
- real,    intent(in)  :: ybeg, yend, ydat(:)
- integer, intent(out) :: js, je
- integer :: j
+! <FUNCTION NAME="get_topog_stdev">
 
-   js = 1
-   do j = 1, size(ydat(:))-1
-      if (ybeg >= ydat(j) .and. ybeg <= ydat(j+1)) then
-         js = j
-         exit
-      endif
-   enddo
```

Comment:
Is this routine actually deleted?  You shouldn't delete any routines

Reply:
It's still there, just outside the ifdef's

https://github.com/rem1776/FMS/blob/5d11c99917dce488a81cd90fc1b40824335dd48d/topography/topography.F90#L1369